app/src/main/java/social/entourage/android/onboarding/login/LoginEmailFragment.kt

```diff
@@ -79,7 +79,7 @@ class LoginEmailFragment : Fragment() {
     //******************************
 
     private fun updateButtonNext() {
-        if (ui_onboard_email_pwd_et_mail?.text!= null && ui_onboard_email_pwd_et_mail.text.toString().isValidEmail()) {
+        if (ui_onboard_email_pwd_et_mail.text.toString().isValidEmail()) {
```

Comment:
ui_onboard_email_pwd_et_mail can be null here

Reply:
Same situation as previous comment l.43